main.tex

```diff
@@ -521,11 +560,118 @@ \subsection{Over-spending prevention by balance proof}\label{balance}
+
+\section{Security and Privacy} \label{sec:securitayandprivacy}
+
+In this section, we discuss the security and privacy guarantees of the WabiSabi credential scheme for construction of CoinJoin transactions. Theft concerns addressed through Bitcoin's security model, making WabiSabi trustless in that regard. Since CoinJoins are an overt technique privacy strongly depends on the structure of the transactions themselves. WabiSabi is designed as a general purpose mechanism, so those details are outside of the scope of this work.
+
+The goal of the protocol is to allow a coordinator to provide the service to honest participants, without learning anything about the mapping between registered input and outputs, apart from what is already deducible given the public amounts visible on the Bitcoin blockchain. WabiSabi leverages the unlinkability of anonymous credentials and the hiding property of the amount commitments to minimize privacy leaks when a set of participants utilizes a centralized coordinator to reach agreement about such a transaction.
+
+\subsection{Availability}
+
+\subsubsection{Malicious Coordinator}
+
+Being a central point of failure, the coordinator is a trusted party with regards to availability. If competing coordinators charge fees for their services then this is a minimal assumption in practice given the financial incentive.
+
+A malicious coordinator can fully disrupt the protocol by censoring certain inputs either at input registration or during the signing phase. Such denial of service can amplify attacks on privacy, by partitioning users in order to perform set intersection attacks. A malicious coordinator can drop messages causing any user to appear to be non-compliant, and therefore disrupt the protocol arbitrarily, and always learns the requested inputs and outputs, even if a round fails.
+
+\subsubsection{Malicious Users}
+Signatures can only be made after a transaction has been negotiated, and all inputs must provide a valid signature. Consequentially users can always disrupt the protocol during the final phase. Failure to sign is attributable to specific inputs and therefore can be mitigated by the coordinator. This allows the remaining honest participants to restart the protocol. Denial of service is not costless because unspent transaction outputs are a limited resource.
+
+A malicious participant still learns all of the input and output registrations\footnote{If \texttt{SIGHASH\_ANYONECANPAY} is set in the signature flags the full set of inputs could be kept known only to the coordinator until all signatures have been provided.} including amounts and scripts, so denial of service also presents privacy concerns.
```

Comment:
Expand on sybil by malicious users to be complete. Make sure that 

- [ ] malicious coordinator availability attacks are discussed - you did this, see my comment above why it's not good
- [ ] malicious coordinator sybil is discussed - you did this, see my comment why I think it's misleading
- [ ] malicious peers availability attacks are discussed - you did this
- [ ] malicious peers sybil is discussed - you didn't do this (well, a few words on spying on incomplete transactions, which is ok, but not complete for full sybils)

Consequently you should fix up the section names.

Reply:
these are both sub-subsections of the "availability" subsection, i.e. denial of service attacks

sybil attacks are covered in the active attacks sub-subsection of the "unlinkability" subsection

Reply:
I resolve my attacks section comments and review it again. I had some confusion, but now more things are clear.